core/modules/ckeditor5/ckeditor5.module

```diff
@@ -4,7 +4,7 @@
  * Provides integration with the CKEditor WYSIWYG editor.
  */
 
-define('CKEDITOR5_VERSION', '44.3.0');
+define('CKEDITOR5_VERSION', '44.1.0');
```

Comment:
Seems like that slipped in by accident - it doesn't belong here. :wink:

Reply:
Correct